src/bb2.py

```diff
@@ -2,13 +2,90 @@
 Blue Button 2.0 SDK Class
 
 """
+import json
+import os
+import pathlib
+import yaml
+
+ROOT_DIR = os.path.abspath(os.curdir) + "/"
+DEFAULT_CONFIG_FILE_LOCATION = ROOT_DIR + "./.bluebutton-config.json"
+
+ENVIRONMENT_URLS = {
+    "SANDBOX": "https://sandbox.bluebutton.cms.gov",
+    "PRODUCTION": "https://api.bluebutton.cms.gov",
+}
 
 
 class Bb2:
     name = "bb2"
     verbose_name = "Blue Button 2.0 SDK Package"
-    
-    def hello(self):
-        mesg = "Hello from BB2 SDK Class method!!!"
-        print(mesg)
-        return mesg
\ No newline at end of file
+
+    def __init__(self, config_file=DEFAULT_CONFIG_FILE_LOCATION):
+        self.config_file = config_file
+        self.config = {}
+
+        self.environment = None
+        self.client_id = None
+        self.client_secret = None
+        self.callback_url = None
+        self.version = 2  # Default to BB2 version 2
+
+        self.base_url = None
+
+        self.config = self._read_config_file()
+
+        # Validate and set
+        self.set_configuration(self.config)
+
+    def _read_json(self, file_path):
+        with open(file_path, "r") as f:
+            data = json.load(f)
+            return data
+
+    def _read_yaml(self, file_path):
+        with open(file_path, "r") as f:
+            return yaml.safe_load(f)
+
+    def _read_config_file(self):
+        extension = pathlib.Path(self.config_file).suffix
+
+        if extension == ".json":
+            return self._read_json(self.config_file)
+        elif extension == ".yaml":
+            return self._read_yaml(self.config_file)
+        else:
+            raise ValueError(
+                "Error: Configuration file extension must be .json"
+                " or .yaml for: {}".format(
+                    self.config_file
+                )
+            )
+
+    def set_configuration(self, config):
+        # Check environment setting
+        env = config.get("environment", None)
```

Comment:
What do you think of defaulting to sandbox?
```suggestion
        env = config.get("environment", "SANDBOX")
```

Reply:
This is matching the new Node SDK behavior where there is an error, if the environment isn't explicitly set to PRODUCTION or SANDOBOX.

Reply:
I guess I was thinking of the case where `environment` is just left off the config. Basically if a value is provided it should be checked against viable values. If it's left off we just default to sandbox. Probably not a big deal but is 1 less thing to get up and running quickly. Only when you transition to prod is it relevant.